wolfcrypt/src/port/caam/caam_qnx.c

```diff
@@ -1190,7 +1190,7 @@ static int getSupported(char* in)
 #endif
 
 char cannedResponse[] = {
-        "wolfCrypt QNX CAAM driver version 4.8.0\n"
```

Comment:
Maybe change this to use `LIBWOLFSSL_VERSION_STRING`?

Reply:
This would be a good change but am trying to keep the driver separated from other code. Eventually I envision it going into its own repo at some point once it gets large enough.... maybe that is the wrong approach and just having the include of version.h here would be better? All of the files with caam_* are part of the driver and all of the wolfcaam_* are part of wolfSSL code interacting with the driver.

Reply:
@JacobBarthelmeh even if it eventually is someplace else you can still include the `wolfssl/version.h` right?

BTW: I like the idea of increasing the version after the final tag.